### PR TITLE
Fix #3534 (Ctrl+R lead to NullReferenceException)

### LINKED ISSUE #3534: Ctrl+R without Ctrl+L raise NullReferenceException
If base commit for compare does not select, Ctrl+R raise NullReferenceException

2017-02-17 14_53_42-gitextensions fix-3534 - git extensions

### PR DESCRIPTION
**Pull request by azarkevich**
Show error if base commit was not selected.

Include Russian translation.
Not sure if English translation correct.

### CODE REVIEW

File: GitUI/Translation/Russian.xlf [hunk 1]
```diff
@@ -9256,6 +9256,11 @@ This will just checkout on the submodule the commit determined by the superproje
         <target>Не возможно подсветить выделенную ветку, когда граф ревизий загружается.</target>
         
       </trans-unit>
+      <trans-unit id="_baseForCompareNotSelectedError.Text">
+        <source>Base commit for compare does not selected.</source>
```
Comment (MarkJacksonFDB) [inline GitUI/Translation/Russian.xlf]: Should be 'is not selected' not 'does not selected'.

File: GitUI/Translation/Russian.xlf [hunk 2]
```diff
@@ -9256,6 +9256,11 @@ This will just checkout on the submodule the commit determined by the superproje
...
+        <target>Не выбран базовый коммит для сравнения.</target>
+
+      </trans-unit>
       <trans-unit id="_droppingFilesBlocked.Text">
         <source>For you own protection dropping more than 10 patch files at once is blocked!</source>
```
Comment (MarkJacksonFDB) [inline GitUI/Translation/Russian.xlf]: Should be 'your own protection' not 'you own protection'.

**Comment by azarkevich** [comment]
English text fixed (push with force)

File: GitUI/Translation/Russian.xlf (hunk 2 above)
Comment (azarkevich) [inline GitUI/Translation/Russian.xlf]: 'For you own protection' - changes not from my commit...

Comment (KindDragon) [inline GitUI/Translation/Russian.xlf]: Please remove this changes from PR. Translation should be done on https://www.transifex.com/git-extensions/git-extensions/

**Comment by azarkevich** [comment]
Russian translation removed

**Pull request merged**